mongo_kerberos.gemspec

```diff
@@ -38,5 +38,5 @@ Gem::Specification.new do |s|
   s.required_ruby_version     = '>= 1.9.3'
   s.required_rubygems_version = '>= 1.3.6'
   s.has_rdoc                  = 'yard'
-  s.add_dependency('mongo', "~> 2.0.0")
+  s.add_dependency('mongo', "~> 2.5.0")
```

Comment:
This can probably stay at 2.0.0. We should test to see if it still works with that driver version but I don't see why not. That way, if someone has the driver installed already, the latest mongo-kerberos can use it.